src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int UPPER_LIMIT = 101;
```

Comment:
what about an access modifier?

Reply:
I didn't get this one, should this be private instead of static?

Reply:
static it os not access modifier. It should be `private static final int`